src/OrleansTestingHost/StorageEmulator.cs

```diff
@@ -190,7 +191,11 @@ private static ProcessStartInfo CreateProcessArguments(string arguments)
         /// <returns></returns>
         private static Process GetStorageEmulatorProcess()
         {
```

Comment:
would submit that and all other StorageEmulator related changes as a separate PR. It is unrelated to this PR and will be lost in this big PR.